core/modules/wiki-bulkops.js

```diff
@@ -13,10 +13,38 @@ Bulk tiddler operations such as rename.
 "use strict";
 
 /*
+Retarget tags and lists from fromTitle toTitle.
+*/
+exports.retargetReferences = function(fromTitle,toTitle) {
```

Comment:
Best to check out my branch and run `bin/server`

[Hub](https://github.com/github/hub) is really helpful for working with Github:

```
hub checkout https://github.com/Jermolene/TiddlyWiki5/pull/2223
```